rules/falco_rules.yaml

```diff
@@ -2566,7 +2566,7 @@
     Bulk data has been removed from disk (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline file=%fd.name container_id=%container.id image=%container.image.repository)
   priority:
     WARNING
-  tags: [process, mitre_persistence]
+  tags: [process, mitre_TA0003_persistence, mitre_TA0005_defense_evasion, mitre_T1070_indicator_removal_on_host]
```

Comment:
the intention of the rule is not to detect `defense evasion`

Reply:
Replaced by `mitre_TA0040_impact` and `mitre_T1485_data_destruction`, if that's ok.